### ISSUE TITLE
CQM Update Procedure Order for CQM models (datetime and reason codes)

### ISSUE THREAD

**Issue by adunsulag** [openemr/openemr#5276]
The procedure_order_code table needs to have UX fields for reason_code, reason_description, reason_date_low, reason_date_high, reason_status.  

The procedure_order table needs to have a datetime selector rather than just a date selector for its UX components.  

We need to update in the system where we create and modify Procedure Orders (and their individual order items) these fields.

**Comment by sjpadgett** [comment]
@adunsulag I don't think we need reason for the order. Don't do anything until we talk. cool?

**Comment by stephenwaite** [comment]
pretty sure we need to add the reason code since I can't get cypress to like my CMS69V10, here's a good result from cypress that have imported into the emr and then re-exported and uploaded into cypress which passes:

![Screenshot from 2022-05-13 09-11-13](https://user-images.githubusercontent.com/5618372/168290592-cdf26a68-ac4f-4fb0-946d-3db3a97a76df.png)

**Comment by stephenwaite** [comment]
passing CMS69V10, thank you @sjpadgett

**Comment by sjpadgett** [comment]
nice work Stephen. Now get back to work!:)

**Comment by stephenwaite** [comment]
hah, funny there are 2 ways to do it and couldn't figure out either, #humbling

**Comment by sjpadgett** [comment]
If I recall there are very few procedures. Mostly care plan and observations

**Comment by stephenwaite** [comment]
turns out intervention performed is one of the few that needs a procedure result since is looking in [procedure_order](https://github.com/openemr/openemr/blob/master/src/Services/Qdm/Services/InterventionService.php#L20-L36)

**Comment by adunsulag** [comment]
So I'm confused, I'm pretty sure I added reason code to both procedures and care plan.  Is the care plan reason code selector not working?

**Comment by stephenwaite** [comment]
trouble was me not using the reason code selector in the procedure order

**Comment by adunsulag** [comment]
So nothing is broken, or is there something I need to fix?

**Comment by stephenwaite** [comment]
it's more user error, the reason code select box does allow you to input a code in an undesired format 

![Screenshot from 2022-05-13 15-57-21](https://user-images.githubusercontent.com/5618372/168380787-6b06dd8f-c453-4e75-b1a1-6c0258c719e4.png)
